superset/advanced_data_type/plugins/eml_path.py

```diff
@@ -0,0 +1,81 @@
+import re
+from typing import Any, List
+
+from sqlalchemy import Column
+from superset.advanced_data_type.types import (
+    AdvancedDataType,
+    AdvancedDataTypeRequest,
+    AdvancedDataTypeResponse,
+)
+from superset.utils.core import FilterOperator, FilterStringOperators
+
+# Operator Sets
+
+EQUAL_NULLABLE_OPERATOR_SET = [
+            FilterStringOperators.EQUALS,
+            FilterStringOperators.NOT_EQUALS,
+            FilterStringOperators.IN,
+            FilterStringOperators.NOT_IN,
+            FilterStringOperators.IS_NOT_NULL,
+            FilterStringOperators.IS_NULL,
+        ]
+
+def translate_filter_func(col: Column, operator: FilterOperator, values: List[Any]) -> Any:
+    """
+    Convert a passed in column, FilterOperator and
+    list of values into an sqlalchemy expression
+    """
+    return_expression: Any
+    if operator == FilterOperator.IN.value:
+        return_expression = col.in_(values)
+    elif operator == FilterOperator.NOT_IN.value:
+        return_expression = ~(col.in_(values))
+    elif operator == FilterOperator.IS_NULL.value:
+        return_expression = col == None
+    elif operator == FilterOperator.IS_NOT_NULL.value:
+        return_expression = col != None
+    elif len(values) == 1:
+        value = values[0]
+        if operator == FilterOperator.EQUALS.value:
+            return_expression = col == value
+        if operator == FilterOperator.NOT_EQUALS.value:
+            return_expression = col != value
+        if operator == FilterOperator.LIKE.value:
```

Comment:
These (LIKE/ILIKE) are not valid operators, we shouldn't get there?

Reply:
Edited in new commit.